visual-tests/test-scripts/series/stacked-bar.js

```diff
@@ -9,17 +9,38 @@
 
     d3.csv('stackedBarData.csv', function(error, data) {
 
-        var categories = data.slice(1).map(function(d) { return d.State; });
-        var yMax = d3.max(data, function(d) {
-            var total = 0;
-            for (var prop in d) {
-                if (prop !== 'State') {
-                    total += Number(d[prop]);
+        var categories = data.map(function(d) { return d.State; });
```

Comment:
I would be tempted to split this into something which [reduced](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce) the array to be grouped by layer (in this case age I think) and then use a [values accessor](https://github.com/mbostock/d3/wiki/Stack-Layout#values) on the stack layout.